README.md

```diff
@@ -386,6 +391,8 @@ Changelog
 [21]: https://cwiki.apache.org/confluence/display/solr/Solr+Plugins
 [22]: https://cwiki.apache.org/confluence/display/solr/Adding+Custom+Plugins+in+SolrCloud+Mode
 [23]: http://www.cominvent.com/
+[24]: https://github.com/healthonnet/hon-lucene-synonyms/releases
+[25]: https://mvnrepository.com/artifact/com.github.healthonnet/hon-lucene-synonyms
```

Comment:
Can we change this link to:

http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.github.healthonnet%22%20AND%20a%3A%22hon-lucene-synonyms%22